crates/kubelet/src/pod/status.rs

```diff
@@ -1,8 +1,28 @@
 //! Container statuses
 
+use super::Pod;
+use crate::container::{make_initial_container_status, ContainerMap, Status as ContainerStatus};
 use k8s_openapi::api::core::v1::ContainerStatus as KubeContainerStatus;
 
-use crate::container::{ContainerMap, Status as ContainerStatus};
+/// Initialize Pod status.
+pub fn make_registered_status(pod: &Pod) -> anyhow::Result<serde_json::Value> {
```

Comment:
This initializes Pod status to include containers in the correct order as expected by `patch_container_status`. It is kind of required, so I'm wondering if this should be done prior to entering the first state handler.

Reply:
As an aside, this comment would be great to add to the doc comments above on line 7.